src/api/routes/v1/validator-report.ts

```diff
@@ -49,10 +49,10 @@ function formatResponse(response: DatabaseResponse): ScoreResponse {
 }
 
 /**
- * Reads nodes from database.
+ * Gets all daily score reports for a validator
  *
- * @param master_key - Master_key of validator.
- * @returns Locations of nodes crawled in the last 10 minutes.
+ * @param master_key - Master key of validator.
+ * @returns A promise that resolves to an array of ScoreResponse 
```

Comment:
Did I miss this comment when initially making it?

Reply:
Or does it run on a new manifest?

Reply:
I'm not sure the comment didn't reflect what the function was doing. It looked like it was copy pasted from the crawler functions. getReports here gets all of the daily score report.